src/main/java/coverage2/FileOutputDemo.java

```diff
@@ -0,0 +1,31 @@
+package coverage2;
+
+import java.io.*;
+
+class FileOutputDemo 
+{   
+
+        public static void main(String args[])
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1197)